JS/Lesson19/gulcin/recursion.js

```diff
@@ -88,10 +88,38 @@ console.log(vowelsInString(sentenceEx4));
 // 4. Write a recursive function to remove all occurrences of a specified character from a string.
 const sentenceEx5 = 'Hello, how are you?';
 const charToRemove = 'o';
+function removeChar(sentence, charToDelete) {
+	if(sentence.length === 0) {
+		return '';
+	} else {
+		if(sentence[0] === charToDelete[0]) {
```

Comment:
Since we need to remove `charToDelete`, you need to compare to it directly. And we also expect it to always be a single letter.
```suggestion
		if(sentence[0] === charToDelete) {
```